src/plugins/unified_search/public/dataview_picker/explore_matching_button.tsx

```diff
@@ -35,7 +35,6 @@ export const ExploreMatchingButton = ({
         alignItems="center"
         gutterSize="none"
         justifyContent="spaceBetween"
-        data-test-subj="select-text-based-language-panel"
```

Comment:
ℹ️ Irrelevant with the ESQL enablement, I just removed the data subj because it is not used and is also wrong